bee-common/bee-common/src/worker.rs

```diff
@@ -20,7 +20,7 @@ pub enum Error {
     #[error("An asynchronous operation failed.")]
     AsynchronousOperationFailed(#[from] std::io::Error),
 
-    /// Occurs, when a message couldn't be sent over a `futures::channel::mpsc` channel.
+    /// Occurs, when a message could not be sent over a channel.
     #[error("Sending a message to a task failed.")]
-    SendingMessageFailed(#[from] futures::channel::mpsc::SendError),
+    SendingMessageFailed,
 }
```

Comment:
I do wonder whether just having this wrap a `Box<dyn Error>` might be more pragmatic. Presumably, none of these errors are meant to be handled and so their only purpose is to be displayable for the user.